### ISSUE TITLE
Update to 1.21?

### ISSUE THREAD

**Issue by aurorarissime** [Leawind/Third-Person#111]

**Comment by aurorarissime** [comment]
Do we have any update, any eta maybe?

**Comment by blepmc** [comment]
Yes please - best third person mod out there we need it on 1.21 / 1.21.1

**Comment by amzelee** [comment]
Is this mod still being worked on?

**Comment by kapuchai** [comment]
Can we have a situation update? Any plans or even thoughts about the port? Nimble got discontinued so this is our last hope

**Comment by amzelee** [comment]
I feel like the author's taking a break from the scene. No updates at all for the last few months. Last I checked on their reddit a week ago, they were quite active in Na Zha reddit threads so I thought they must be enjoying their time off modding. I'm personally just hoping they come back soon when they feel like it. If not, then maybe someone can make a port of the mod to higher versions.

**Comment by kapuchai** [comment]
![Image](https://github.com/user-attachments/assets/c71fd40f-6e43-4b11-a754-f5403e7b76d1)

**Comment by kapuchai** [comment]
Or at least I'll try lol, looking at the code it doesn't seem to be that complicated

**Comment by amzelee** [comment]
Damn, update us when you do. I'll be looking forward to it.

**Comment by kapuchai** [comment]
Not making any promises though. While I'm still looking into the code so far its fascinating. Its very organized and clean, author did an amazing job here

**Comment by LOLZSICKA** [comment]
went to install for 1.21.4 client and found it does not exist, sad

**Comment by amzelee** [comment]
Yeah seems the mod is abandoned. I come back here every now and then in hopes that the author looks into it and ports. Or maybe someone ports an unofficial one.

**Comment by kapuchai** [comment]
According to recent git commits it looks like the author is back and started actively porting the mod to versions 1.21-1.21.2 in April!

**Comment by Lumiko32** [comment]
It's exciting news, and I'm keeping an eye on it

**Comment by enzogfaria** [comment]
the mod is working on 1.21.1 guys, u just need to build the version that is on github

**Comment by aaschram** [comment]
> the mod is working on 1.21.1 guys, u just need to build the version that is on github

Can you send a download link of the version you built? I tried to put together a folder of all the updated files and convert it to a jar file, but that didn’t work out for me
